projects/kit/components/text-area/text-area.component.ts

```diff
@@ -42,6 +42,9 @@ export const DEFAULT_ROWS = 20;
 export const LINE_HEIGHT_M = 20;
 export const LINE_HEIGHT_L = 24;
 
+const DEFAULT_HEIGHT_REM = 6.75;
+const DEFAULT_HEIGHT_COUNTER_REM = 7.875;
+
```

Comment:
should this code be moved to DI options?

Reply:
This is not configurable, it's just for calculations without magic numbers